nntrainer/layers/lstm.cpp

```diff
@@ -153,7 +170,22 @@ void LSTMLayer::forwarding(bool training) {
   Tensor &bias_h =
     weightAt(static_cast<int>(LSTMParams::bias_h)).getVariableRef();
 
-  Tensor &hidden_ = net_hidden[0]->getVariableRef();
+  mem_cell->getVariableRef().setZero();
+  hidden->getVariableRef().setZero();
```

Comment:
some of the setZero are redundant here. Add a TODO to update them later?

Reply:
No. it is necessary for iteration.

Reply:
given line 237L, i dont think setZero is needed.

Reply:
This must be zero for next iteration when it starts.

Reply:
its fine for now, this can be done later when we optimize rnn/lstm implementations.